go.mod

```diff
@@ -1,21 +1,24 @@
 module github.com/ObolNetwork/lido-dv-exit
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
```

Comment:
why do we need a stricter toolchain version here?

Reply:
probably due to https://go.dev/doc/toolchain

it appeared after a `go mod tidy`